core_backend/app/question_answer/routers.py

```diff
@@ -276,9 +276,6 @@ async def search_base(
         If the question language is not identified.
     """
 
-    if query_refined.original_language is None:
```

Comment:
Removed this code since we do the checks in guardrails.

Reply:
Can you update the docstring for `async def search_base` to remove the section:

```
    Raises
    ------
    ValueError
        If the question language is not identified.
```